css-layout-classes/styles.css

```diff
@@ -0,0 +1,17 @@
+.container {
+  width: 1000px;
+  margin: auto;
+  font-family: sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-fourth {
+  max-width: 250px;
```

Comment:
instead of max-width, take advantage of `flex-basis` on your column classes to ensure they stay within the row.